### ISSUE TITLE
InvalidMixinException when running 22w11a

### ISSUE THREAD

**Issue by Olexorus** [gnembon/fabric-carpet#1367]
When running the latest Carpet version for 22w11a, I get this error six times:
```
[09:06:22] [main/INFO]: Loading Minecraft 22w11a with Fabric Loader 0.13.3
[09:06:22] [main/INFO]: Loading 4 mods:
	- carpet 1.4.67
	- fabricloader 0.13.3
	- java 17
	- minecraft 1.19-alpha.22.11.a
[09:06:22] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/home/oliver/MinecraftWorkspace/SnapshotTestServer/libraries/net/fabricmc/sponge-mixin/0.11.2+mixin.0.8.5/sponge-mixin-0.11.2+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[09:06:22] [main/INFO]: Compatibility level set to JAVA_16
[09:06:25] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[09:06:28] [Worker-Main-24/ERROR]: Mixin apply for mod carpet failed carpet.mixins.json:TagPredicate_scarpetMixin -> net.minecraft.class_2252$class_2255: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow field tag was not located in the target class net.minecraft.class_2252$class_2255. Using refmap fabric-carpet-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field tag was not located in the target class net.minecraft.class_2252$class_2255. Using refmap fabric-carpet-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachFields(MixinPreProcessorStandard.java:624) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:302) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1289) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:292) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_2252.method_41952(class_2252.java:51) ~[server-intermediary.jar:?]
	at com.mojang.datafixers.util.Either$Right.map(Either.java:99) ~[datafixerupper-4.1.27.jar:?]
	at net.minecraft.class_2252.method_41953(class_2252.java:49) ~[server-intermediary.jar:?]
	at net.minecraft.class_2252.method_9642(class_2252.java:45) ~[server-intermediary.jar:?]
	at net.minecraft.class_2252.parse(class_2252.java:30) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.ArgumentCommandNode.parse(ArgumentCommandNode.java:59) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:365) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:392) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:392) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:392) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:388) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:392) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parseNodes(CommandDispatcher.java:392) ~[brigadier-1.0.18.jar:?]
	at com.mojang.brigadier.CommandDispatcher.parse(CommandDispatcher.java:348) ~[brigadier-1.0.18.jar:?]
	at net.minecraft.class_2158.method_9195(class_2158.java:56) ~[server-intermediary.jar:?]
	at net.minecraft.class_5349.method_29451(class_5349.java:87) ~[server-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
```
I redownloaded Fabric to make sure I have the latest version, and I don't have any other mods installed.

**Comment by liquiddevelopmentnet** [comment]
same for me

**Comment by James103** [comment]
Running the command `fill ~ ~ ~ ~ ~1 ~ air replace #logs` errors with "Could not parse command" followed by the above exception, regardless of how the command is run. The same command with a different tag after `replace` fails with the same error, but individual blocks such as `oak_log` work.

**Comment by Olexorus** [comment]
Seems to be fixed now, 1.4.68 works on 22w12a for me
